obspy/io/css/core.py

```diff
@@ -119,3 +151,59 @@ def _read_css(filename, **kwargs):
         tr = Trace(data, header=header)
         traces.append(tr)
     return Stream(traces=traces)
+    
+def _read_nnsa_kb_core(filename, **kwargs):
+    """
+    Reads a NNSA KB Core waveform file and returns a Stream object.
+    .. warning::
+        This function should NOT be called directly, it registers via the
+        ObsPy :func:`~obspy.core.stream.read` function, call this instead.
+    :type filename: str
```

Comment:
Please add blank lines after first line and before parameter descriptions, to be in line with the rest of our codebase.